index.js

```diff
@@ -1,5 +1,67 @@
 'use strict';
 
+var superagent = require('superagent');
+
+global.D2LAccessTokenExpiresAt = 0;
+
+function now() {
+	return Date.now()/1000 | 0;
+}
+
+function addHeaders(req) {
+	req.set('X-Csrf-Token', localStorage['XSRF.Token']);
+	req.set('X-D2L-App-Id', 'deprecated');
+}
+
+function processRefreshResponse(cb) { return function(res) {
+	if (!res.ok) {
+		return cb(false);
+	}
+
+	var cacheControl = res.headers['cache-control'];
+	if (!cacheControl) {
+		return cb(true);
+	}
+
+	var directives = cacheControl.split(',');
+	for (var i = 0; i < directives.length; i++) {
```

Comment:
I know it's mostly meaningless because `i` will basically always be `1`, but better to cache `arr.length` into a variable, generally speaking